roles/rhods/rhods_deploy_ods/templates/catalog.yml.j2

```diff
@@ -2,11 +2,13 @@ apiVersion: operators.coreos.com/v1alpha1
 kind: CatalogSource
 metadata:
   name: {{ rhods_catalog_name }}
-  namespace: {{ rhods_namespace }}
+  namespace: openshift-marketplace
 spec:
   displayName: {{ rhods_catalog_display_name }}
   image: {{ rhods_deploy_ods_catalog_image }}:{{ rhods_deploy_ods_tag }}
   publisher: {{ rhods_catalog_publish }}
   sourceType: grpc
+  grpcPodConfig:
+    securityContextConfig: restricted
```

Comment:
nice :)

Reply:
This is needed on latest OCP version AFAICT